server/internal/mcp/rpc_tools_list.go

```diff
@@ -76,6 +77,15 @@ func handleToolsList(
 		fallthrough
 	default:
 		tools = buildToolListEntries(toolset.Tools)
+
+		// Unfold proxy tools from external MCP servers
+		unfoldedTools, err := unfoldExternalMCPTools(ctx, logger, toolset.Tools, payload.oauthTokenInputs)
```

Comment:
So checking my understanding here. It seems like we actually only unfold an external tool in mcp context. So right now these tools wouldn't be visible/useable in the playground. Correct?